oio/container/app.py

```diff
@@ -424,7 +441,8 @@ def _do_head(self, _, account, container):
             'X-Blocks': sum([i['block'] for i in results]),
             'Content-Length': sum([i['block'] for i in results]) * BLOCKSIZE,
             'Accept-Ranges': 'bytes',
-            'Content-Type': 'application/tar'
+            'Content-Type': 'application/tar',
+            'X-Manifest-Container': json.dumps(results),
```

Comment:
Agreed, I will store the manifest as special file like it was done for container properties to avoid this kind of issue.